compiler/common-artifacts/exclude.lst

```diff
@@ -5,6 +5,8 @@
 
 #[[ optimize : Exclude from circle optimization(circle2circle) ]]
 ## TensorFlowLiteRecipes
+optimize(UnidirectionalSequenceLSTM_000) # This will be removed after all luci components done.
```

Comment:
Does `UnidirectionalSequenceLSTM_000` contain things `circle2circle` cannot run like `is_variable`? AFAIK, it doesn't need to add it to `optimize` exclude list if related PRs are merged proper order.

Reply:
No, it is model that composed of all weight, bias and etc as input. If I add recipe at the last part, those line is not needed. I will remove that and add recipe on last part(after luci component add.)